### DISCUSSION TITLE
Wierd thing happened.

### DISCUSSION THREAD

**Discussion by ProdanAS** [john30/ebusd#1203] [Q&A]
Yesterday, I updated my ebusd installation, and removed my local Valiant repository, and ran on the standard GIT. 

When I woke up this morning, the HeatPump was stopped, and the water cold. I could see that the compressor had started like 30 times during the night, and shut off emidiatly again. 

I had my plumber out to inspect why, and he couldn't see any problems, but discovered this happening. 

The Flow Calculated temp was at 60 degrees, when the compressor started, this changed to 22, and the compressor stopped, then it changed to 60 degrees again. 

As i understand it, it's Read Only, so it shouldnt write values into the Valiant control. But the fun thing was, that when I shut down ebusd, the HP resumed normal working conditions. 

My question here is, WTF, how can this happen. I'm totally at loss of ideas.

**Comment by ProdanAS** [comment]
<img width="638" alt="Skærmbillede 2024-02-22 kl  15 13 49" src="https://github.com/john30/ebusd/assets/116255260/7e97da62-3ebd-4d7f-b59f-a8bb6a6d7642">
<img width="539" alt="Skærmbillede 2024-02-22 kl  15 14 03" src="https://github.com/john30/ebusd/assets/116255260/4ac7bb32-b65d-44b2-bc39-a687858aaf91">
<img width="605" alt="Skærmbillede 2024-02-22 kl  15 14 08" src="https://github.com/john30/ebusd/assets/116255260/fb13c50b-f533-4a92-b88d-fef109d435f2">
<img width="481" alt="Skærmbillede 2024-02-22 kl  15 14 32" src="https://github.com/john30/ebusd/assets/116255260/257f72f6-57d3-41e4-8167-248f59882aaf">

**Reply by ProdanAS** [reply]
<img width="563" alt="Skærmbillede 2024-02-22 kl  15 16 21" src="https://github.com/john30/ebusd/assets/116255260/97756039-b101-4fb3-93ad-2fe04dcbae51">
<img width="568" alt="Skærmbillede 2024-02-22 kl  15 16 15" src="https://github.com/john30/ebusd/assets/116255260/8a69ebdc-144c-47d2-825c-9f9b86c058e1">

**Comment by JonesGW-MGD** [comment]
Can you share the logs?  Ignoring whether configuration is readable or writable, Do you have any HA automations that is configured to update ebusd?

If you have a hypothesis that the update to ebusd and the erratic behaviour of your HP are not a coincidence, then you should be able to find some evidence for this in ebusd/HA/MQTT logs, correct?  The nice thing about software is it generally leaves a trail!!

**Reply by ProdanAS** [reply]
I will see if i can grap the logs tomorrow. It's plain standard docker image. I've modiefied the hassio CFG to send pressure as well.  All other are default. Im using mqtt to publish to home assistant. There's no configuration or anything setup here. It has been running with the same configuration for 3 weeks. Until i remove the enviorment variable last night containing the custom csv files. Ill return tomorrow with the docker logs

**Reply by ProdanAS** [reply]
This is from today, the HP stopped after a few minutes upon starting the container, this is the complete log, but stopped when the HP reached 0 power usage. 
[john30-ebusd1.csv](https://github.com/john30/ebusd/files/14382397/john30-ebusd1.csv)

**Reply by ProdanAS** [reply]
![Skærmbillede 2024-02-23 kl  07 57 06](https://github.com/john30/ebusd/assets/116255260/da0efe4f-1b27-4ce7-ac4b-bfdc544bd2bf)

**Reply by JonesGW-MGD** [reply]
How odd.  Bearing in mind I'm fairly new to ebusd myself and not a HP expert so hopefully someone else will weigh in.

The logs look like they have very little traffic between the ebusd master do they? - (mostly making reads and a few unknown commands - 3108b5090124 / 09003231323233373030 and 3176b5090124 / 09003231323233373030 - where it gets back the same response from both HP master addresses. 

Quite a lot of traffic between what I assume are you indoor and outdoor components (guessing maybe you have an Aerotherm Split?).  0376... and 7108.... but it's all un decoded messages unfortunately.  

A couple of 'stupid' questions (sometimes helps, often not!).  First, are you in a position to re-instate the local config files and if so does the HP work fine again?   Second, (and I think you let your heating engineer off lightly!). But did he not offer any explanation for the symptoms of what you experience (rather than the cause). 

Because it certainly isn't true that he 'couldn't see any problems' unless you had ebusd switched off then?  I ask because I wonder what the first thought of an engineer might have been if he saw your specific behaviour but didn't know you had some magic also hooked up?

I assume that either you or the engineer can check the error codes physically on the device and there isn't anything remiss?

I hope you can get it sorted; I'm interesting in your outcome, good luck!

**Reply by ProdanAS** [reply]
I will try to reinstate the other config. See if there's a difference, wich I presume. 

It a aerotherm monoblock, adressed by a VWZ AI. The Plumber is a friend of mine, and he knows that im running the ebusd system. So when he spotted the odd switch in values, I asked him to not do anymore until I've analysed this behaviour. 

The problems referred to was wierd sounds, leaking pipes ect. 

Attached is the new log with the Old files :) From timestamp 13.50 
[john30-ebusd1.csv](https://github.com/john30/ebusd/files/14385909/john30-ebusd1.csv)

The HP did not shut down with these files 
[EbusConfig.zip](https://github.com/john30/ebusd/files/14385913/EbusConfig.zip)

**Reply by JonesGW-MGD** [reply]
I notice you had a specific file for the HP locally. - 76.vwz.csv (although it seemed you had some issues and maybe it never use to load properly anyway.  That doesn't seem to exist in the repository anymore - maybe it's been merged into 08.hmu.csv and there is some odd behaviour as a result

**Comment by KnatterKnilch** [comment]
Hi, I experienced the exactly same issue here:
1. HP System works fine for > 1 Month with the following setup: Vaillant Arotherm VWL Plus 75/6 with indoor unit, heating controller and proprietary internet gateway. Integrated into Home Assistant via eBUS Adapter Shield v5 and eBusd HA container 
2. On Feb. 14th suddenly the HP starts to shut down after ~2min of operation -> going into standby -> re-try with same outcome. Thus the oscillation on the flow temperature...
3. Issue disappears as soon as ebusd is started in --readonly mode.

unfortunately I wasn't able to recover the full logfile out of the HAs ebusd docker container but I noticed something else which also fits your timing: There was a commit to the relevant ebusd configuration file ebusd-2.1.x/de/vaillant/08.hmu.csv on Feb 11th: [9909b3a](https://github.com/john30/ebusd-configuration/commit/9909b3a34d6d5fb646f94dde1b5df308f0a295bb)

Any idea what change exactly could have caused that behavior?

**Reply by JonesGW-MGD** [reply]
Only because I now have an interest I had a look - seems the update to be work to incorporate LiveMonitor settings?  https://github.com/john30/ebusd-configuration/pull/316 Seems though that to read live monitor settings you need to write first.  Maybe for certain configurations that causes some kind of system 'panic'?  I don't understand why if you had no explicit write requests from your ebusd setup it would do that though - above my ebusd 'paygrade'.

What I would say is you are not the only ones - before the merge, this comment - https://github.com/john30/ebusd-configuration/pull/316#issuecomment-1902784959

Seems like someone else @Dzanar experienced issue like you're experiencing in January but his comment did not look like it was followed up.   The update was also apparently a combination of another stream of work/pull request [330](https://github.com/john30/ebusd-configuration/pull/330) - maybe a starting point for you both is contacting some of the folks active on those threads. As always good luck (and it's always good to hear updates on these threads out of interest - so many end cold).

**Reply by JonesGW-MGD** [reply]
Also: May not be the same issue but another heat pump user:  https://github.com/john30/ebusd/issues/1205
